### PR TITLE
Pr@dev@wjh

### PR DESCRIPTION
**Pull request by ziyujiahao**

### CODE REVIEW

**Comment by ziyujiahao** [comment]
feat: 仪表板增加富文本编辑

File: frontend/public/tinymce/langs/zh_TW.js [hunk 1]
```diff
@@ -0,0 +1,419 @@
+tinymce.addI18n('zh_TW',{
+"Redo": "\u91cd\u505a",
```
Comment (fit2cloudrd) [inline frontend/public/tinymce/langs/zh_TW.js:2]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687731)
```suggestion
  "Redo": "\u91cd\u505a",
```

File: frontend/public/tinymce/langs/zh_TW.js [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+"Copy": "\u8907\u88fd",
+"Paste": "\u8cbc\u4e0a",
+"Select all": "\u5168\u9078",
+"New document": "\u65b0\u6587\u4ef6",
+"Ok": "\u78ba\u5b9a",
+"Cancel": "\u53d6\u6d88",
+"Visual aids": "\u5c0f\u5e6b\u624b",
+"Bold": "\u7c97\u9ad4",
+"Italic": "\u659c\u9ad4",
+"Underline": "\u4e0b\u5283\u7dda",
```
Comment (fit2cloudrd) [inline frontend/public/tinymce/langs/zh_TW.js:14]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687731)
```suggestion
'Underline': "\u4e0b\u5283\u7dda",
```

File: frontend/public/tinymce/langs/zh_TW.js [hunk 3]
```diff
@@ -0,0 +1,419 @@
+tinymce.addI18n('zh_TW',{
+"Redo": "\u91cd\u505a",
+"Undo": "\u64a4\u92b7",
+"Cut": "\u526a\u4e0b",
+"Copy": "\u8907\u88fd",
+"Paste": "\u8cbc\u4e0a",
+"Select all": "\u5168\u9078",
+"New document": "\u65b0\u6587\u4ef6",
```
Comment (fit2cloudrd) [inline frontend/public/tinymce/langs/zh_TW.js:8]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687731)
```suggestion
'New document': "\u65b0\u6587\u4ef6",
```

**Pull request closed without merging**